helpers/GSGetter.ts

```diff
@@ -257,18 +287,46 @@ export class GCGetter {
                 for (let index = 0; index < all_users_id.length; index++) {
                     email_ids[index] = all_users_id[index].emails[0].address;
                 }
-                await modify.getCreator().finish(message);
                 let invite_token = await persistence.get_access_token(context.getSender());
                 const invite_parameters = context.getArguments().join(' ');
                 const invite_array = invite_parameters.split("\"");
                 const invite_url = `https://www.googleapis.com/calendar/v3/calendars/primary/events?key=${api_key}&sendUpdates=all`;
 
-                const invite_datetime = invite_array[3] + 'T' + invite_array[5];
+                const invite_datetime = invite_array[3] + 'T' + invite_array[5] + 'Z';
                 const invite_date = new Date(invite_datetime);
                 const invitestart_datetime = invite_date.toISOString();
-                const invite_e_date = invite_array[3] + 'T' + invite_array[7];
+                const invitestart_ms = invitestart_datetime.split(".");
+                const invite_e_date = invite_array[3] + 'T' + invite_array[7] + 'Z';
                 const inviteend_date = new Date(invite_e_date);
                 const invite_end_datetime = inviteend_date.toISOString();
+                const inviteend_ms = invite_end_datetime.split(".");
+
+                const users_info = await read.getUserReader().getById(users_id);
+
+                let final_start;
+                let final_end;
+                let utcoffset = users_info.utcOffset;
+                let decimal_part = Math.abs(utcoffset - Math.floor(utcoffset));
+                decimal_part = decimal_part * 60;
+                if (utcoffset > 0) {
```

Comment:
This looks very similar to the code above, can you extract this logic into a function?

Reply:
https://github.com/RocketChat/Apps.Google.Calendar/blob/Fix2/helpers/result.ts#L68
Made the required changes.